students/ghassan/lesson03/slicing.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+
+def first_last(data):
+    new_list = data[:]
+    new_list[0] = data[-1]
+    new_list[-1] = data[0]
+    return new_list
+
+
+def every_other(data):
+    new_list = data[::2]
+    return new_list
+
+
+def fourth_fourth(data):
+    new_list = data[0:4] + data[-4:]
+    return new_list
+
+
+def reversed(data):
+    return data[::-1]
+
+
+def thirds(data):
+    first_third = int(len(data) / 3)
+    middle_third = int((len(data) / 3) * 2)
+    last_third = data[-1]
+    return [data[middle_third], last_third, first_third]
```

Comment:
so given input `data = (2, 54, 13, 12, 5, 32)`, your expected data is: `(13, 12, 5, 32, 2, 54)`

you would divide your input into 3 parts, then return middle part + last part + first part